### PR TITLE
Minify & Concat Devices

### PR DESCRIPTION
**Pull request by adamjc**
🤖

### CODE REVIEW

**Comment by adamjc** [comment]
Uh -- wait until I validate output against schema guys...

File: scripts/concat-devices.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const glob = require('glob')
+const path = require('path')
+
+const filename = process.argv[2]
```
Comment (AlexChesters) [inline scripts/concat-devices.js]: I might be being a bit pedantic but I think `filename` should be named `directory` or something similar.

Comment (adamjc) [inline scripts/concat-devices.js]: Ah yes, shallow and pedantic

**Comment by adamjc** [comment]
Actually, I'll validate afterwards -- need to update the schema validation to validate filenames anyway I think...

**Pull request merged**